scripts/jenkins/ardana/ansible/roles/heat_stack/defaults/main.yml

```diff
@@ -17,7 +17,8 @@
 
 heat_action: "create"
 
-heat_stack_create_timeout: 900
+# The timeout used by heat for the create/delete stack operations
+heat_stack_timeout: 480
```

Comment:
Isn't safer to have 900 as def. value in the role. Since we have susecloud with 900 and maybe when running manually `heat_stack_timeout` var will not be setup.

Reply:
@djz88 very good point, I completely overlooked the manual use-case. Thanks !

Reply:
done